docker-compose.yml

```diff
@@ -1,8 +1,6 @@
-version: "3"
-
 services:
   kutt:
-    image: kutt/kutt
```

Comment:
@hammady Not a review comment, is this why testing docker compose locally didn't work?

Reply:
Yes, sir :)